src/VuetifyFileManager.vue

```diff
@@ -88,14 +86,17 @@ export default {
     },
     methods: {
         folderSelected(selectedFolderId) {
-            this.getFolderContent(selectedFolderId).then((content) => {
+            this.selectedFolder = selectedFolderId;
```

Comment:
I have trust issues with this variable name.

on the RHS you suffix it with `id`, but on the LHS, you do not.

I would expect the name `selectedFolder` to contain an object instead of a number.